blogposts/2020/evolution-of-the-precise-code-intel-backend.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Evolution of the precise code intel backend"
+description: This post reflects on the high-level technical changes as the precise code intel services providing the feature matured through additional features, changing environment requirements, hardening, performance improvements, refactoring, and one major rewrite in a different language. 
```

Comment:
"providing the feature ... additional features"  repeats the word features. 

i would say: "providing code intelligence"

Reply:
Updated.